src/plugins/intel_cpu/src/transformations/snippets/x64/pass/eliminate_brgemm_copy_b.cpp

```diff
@@ -36,20 +43,34 @@ pass::EliminateBrgemmCopyB::EliminateBrgemmCopyB() {
             return false;
         }
 
+        // Update external repacking config for the further pipeline stages
+        const auto& param = ov::as_type_ptr<ov::op::v0::Parameter>(pattern_map.at(m_param).get_node_shared_ptr());
+        const auto param_idx = static_cast<size_t>(model->get_parameter_index(param));
+        OPENVINO_ASSERT(param_idx < model->get_parameters().size(),
+                        "Parameter index is invalid in EliminateBrgemmCopyB transformation");
+        // Mark this input as repacked
+        auto& config = m_constant_inputs_idxs.count(param_idx) == 0 ? m_repacked_runtime_inputs_config
+                                                                    : m_repacked_constant_inputs_config;
```

Comment:
Maybe we can save some status (runtime/constant repacking) in `RepackedInput` member instead of separating them into 2 different configs? What do you think?

Reply:
Discussed offline: current differentiation on 2 configs is justified